lib/smart_proxy_remote_execution_ssh/plugin.rb

```diff
@@ -2,7 +2,10 @@ module Proxy::RemoteExecution::Ssh
   class Plugin < Proxy::Plugin
     SSH_LOG_LEVELS = %w[debug info error fatal].freeze
     MODES = %i[ssh async-ssh pull pull-mqtt].freeze
-    SOCKET_PATH_MAX_LENGTH = 18.freeze
+    # Unix domain socket path length is limited to 104 (on some platforms) characters
+    # Socket path is composed of custom path (max 49 characters) + job id (37 characters)
+    # + offset(17 characters) + null terminator
```

Comment:
What is the offset?

Reply:
According to `man 7 unix` it should be the size of the address structure also known as `offsetof(struct sockaddr_un, sun_path)`, where `sun_path` is socket pathname. Maybe `sizeof sockaddr` would be better name?

Reply:
No, it is fine.